website/assets/js/pages/customers/new-license.page.js

```diff
@@ -69,9 +77,7 @@ parasails.registerPage('new-license', {
         this.formData.quoteId = quote.id;
         this.showBillingForm = true;
       }
-      // When the final submit has been rendered into existence, focus it for our friendly user.
       await this.forceRender();
-      this.$focus('[purpose="submit-button"]');
```

Comment:
?

Reply:
This was a request from @mike-j-thomas while we were taking a look at this together on a call.

@mike-j-thomas Do you remember why we made this change?

Reply:
@eashaw, was this line what was causing the page to focus on the billing and submit button after the user entered their number of devices? If so, the jump in focus was disorientating, so I asked for the page not to move. Cc @mikermcneil

Reply:
Gotcha.  In the future, I'd recommend doing that as a separate change.  Makes the PR easier to review.